spec/features/01_spells_pagination_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "spells index page paginates all spells" do
+  before(:all) do
+    51.times do
+      FactoryGirl.create(:spell)
```

Comment:
This is going to make one call per create. Factory girl supports `create_list` method which takes the :spell and a number you want created

Reply:
I also am not sure we need to test this bit. We can rely on the authors of kaminari testing their own gem well enough

Reply:
Yeah, that's true. I had created it just to be sure that the index was displaying correctly, but we probably don't need it. Removed.